doc/examples/filters/plot_enhancemet_measure.py

```diff
@@ -0,0 +1,65 @@
+"""
+=========================================
+Quantitative Measure of Image Enhancement
+=========================================
+
+In this example, we will measure the result of applying contrast enhancement
+to the image. We will use a measure called EME [1]_.
+
+It is defined as a log of the ratio of local maximum to local minimum
+within a sliding window. The code allows the user to choose the window size.
+This function was defined to quantify improvement of the image after processing.
+
+"""
+
+import numpy as np
+import matplotlib
+matplotlib.use("TkAgg")
```

Comment:
These two lines above are not needed and should be removed.

Reply:
Thank you for your comments! I fixed the issues.